packages/@aws-cdk/aws-stepfunctions/test/fields.test.ts

```diff
@@ -18,6 +18,18 @@ describe('Fields', () => {
             deepField: JsonPath.numberAt('$.numField'),
           },
         ],
+        deepList: [
+          [
+            [
+              [
+                'literal',
+                {
+                  deepField: JsonPath.numberAt('$.numField'),
+                },
+              ],
+            ],
+          ],
+        ],
```

Comment:
we typically do not modify existing tests as it may mask a regression / have an unintended side effect. I think it's fine for this specific case however.